tests/test_freqtradebot.py

```diff
@@ -238,143 +238,143 @@ def test_edge_overrides_stoploss(limit_buy_order, fee, caplog, mocker,
         assert trade.sell_reason == SellType.STOP_LOSS.value
 
 
-def test_total_open_trades_stakes(mocker, default_conf, ticker, fee) -> None:
+def test_total_open_trades_stakes(mocker, default_conf_usdt, ticker_usdt, fee) -> None:
     patch_RPCManager(mocker)
     patch_exchange(mocker)
-    default_conf['stake_amount'] = 0.00098751
-    default_conf['max_open_trades'] = 2
+    default_conf_usdt['stake_amount'] = 10.0
+    default_conf_usdt['max_open_trades'] = 2
     mocker.patch.multiple(
         'freqtrade.exchange.Exchange',
-        fetch_ticker=ticker,
+        fetch_ticker=ticker_usdt,
         get_fee=fee,
         _is_dry_limit_order_filled=MagicMock(return_value=False),
     )
-    freqtrade = FreqtradeBot(default_conf)
+    freqtrade = FreqtradeBot(default_conf_usdt)
     patch_get_signal(freqtrade)
     freqtrade.enter_positions()
     trade = Trade.query.first()
 
     assert trade is not None
-    assert trade.stake_amount == 0.00098751
+    assert trade.stake_amount == 10.0
     assert trade.is_open
     assert trade.open_date is not None
 
     freqtrade.enter_positions()
     trade = Trade.query.order_by(Trade.id.desc()).first()
 
     assert trade is not None
-    assert trade.stake_amount == 0.00098751
+    assert trade.stake_amount == 10.0
     assert trade.is_open
     assert trade.open_date is not None
 
-    assert Trade.total_open_trades_stakes() == 1.97502e-03
+    assert Trade.total_open_trades_stakes() == 20.0
```

Comment:
For some reason `Trade.total_open_trades_stakes()` evaluates to 10.0, and when I check `Trade.query.all()` I get an array with only one trade in it `[Trade(id=1, pair=ETH/USDT, amount=5.00000000, open_rate=2.00000000, open_since=2021-09-26 20:10:16)]`

Reply:
I'm guessing that this is because I need to add data to `tests/testdata`, I don't really know what's the best way to do this